aries-site/src/pages/components/namevaluelist.mdx

```diff
@@ -0,0 +1,232 @@
+    code={["https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/namevaluelist/NameValueListGridExample.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/namevaluelist/data.js",
+]}
+  height={{ min: 'small' }}
+  template
+>
+  <NameValueListGridExample />
+</Example>
+
+## Hierarchy
+
+A successful NameValueList is scannable and easy to grasp. Consider the following methods 
+to imply hierarchy: [order](#order), [headings](#headings), [scale](#scale), 
+[font weight](#font-weight), [color](#color).
+
+### Order
+
+- Having a logical order of a name-value list ensures a more scannable display of data.
+- Depending on content displayed, consider ordering by name (alphabetical order) or by importance of information (most relevant to less relevant).
+- Always establish the structure of your data upfront.
+
+### Headings
+
+- Use headings to define a Name-Value List and/or for grouping multiple name-value lists within a single screen.
+- For accessibility reasons, always ensure semantically correct heading levels are applied. See [best practices for applying headings](/foundation/typography#semantic-usage-of-heading-levels).
+
+ <Example content="Example: An h2 small heading is applied to a vertical name-value list" pad="none" height={{ min: 'small' }}>
+  <NameValueListHeadingPreview />
+ </Example>
+
+### Scale
+
+- Default font size for a NameValueList is “medium” (18px).
+- Font-size options for displaying name and value include small, medium and large.
+- Additional sizes xl and xxl can be opted for displaying value content, depending on styling needs (such as for displaying on a dashboard page layout).
+- The size of name can differ from value.
+- Name should never be scaled bigger than its value.
+
+<NameValueListScaleTable />
+
+<BestPracticeGroup>
+  <Example
+    height="none"
+    bestPractice={{type: 'do', message: 'In this example, names are listed in xsmall size with values displayed in default font-size.'}}
+  >
+    <NameValueListTextGoodExample />
+  </Example>
+  <Example
+    height="none"
+    bestPractice={{type: 'dont', message: `Don’t scale names bigger than their value.`}}
+  >
+    <NameValueListTextBadExample />
+  </Example>
+</BestPracticeGroup>
+
+### Font weight
+
+Making the font bolder or thinner is another way to create hierarchy that’s easily identifiable.
+
+- Typically, use a bold weight on names to increase scanability within the list
+- In some cases, a value may be displayed in a bold weight to draw the reader’s attention. An example of this application might be a dashboard page.
+- Don’t use bold text for both name and value at the same time.
+- Don’t use a bold font-weight for any long format content as this can be very hard to read and overwhelm the user.
+
+  <Example caption="Example: A two column Name-Value List where the names are in a bold font-weight" pad="none" height="none">
```

Comment:
We should adjust references of "two column" and "multi-column" to match whatever is resolved from the slack poll regarding "Vertical, Horizontal, Grid" etc

Reply:
I'll wait till EOD than implement this